tests/core_tests/test_log.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) 2017 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+import os
+import copy
+
+import sgtk
+from tank_test.tank_test_base import TankTestBase
```

Comment:
Does the test really pass? I seem to recall tests failing if `setUpModule` is not called. Maybe I am wrong?

Reply:
Weird...you're right that it fails when I run test_log.py only. Importing * sorts it out, though that's going to raise the ire of hound. We can ignore that in this case, though.